tools/javadoc-prettifier/build.gradle

```diff
@@ -24,6 +24,12 @@ apply plugin: 'java-gradle-plugin'
 dependencies {
     implementation deps.build.guava
 
-    implementation project(':plugin-base')
     implementation project(':base')
+    implementation project(':plugin-base')
+
+    testImplementation project(':plugin-base').sourceSets.test.output // Test classes of `plugin-base`
```

Comment:
Let's try to move the required classes to some test utility module instead of reusing the compile output of the test sources.